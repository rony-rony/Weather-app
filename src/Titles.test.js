import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import Titles from './components/Titles';

it('renders Titles wrapper without crashing', () => {
    const wrapper = shallow(<Titles />);
    expect(wrapper).toMatchSnapshot();
});

it('the type ot titles child', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Titles />);
    const result = renderer.getRenderOutput();

    expect(result.type).toBe('div');
    expect(result.props.children).toEqual([
        <h1 className="title-container__title">Weather Finder</h1>,
        <p className="title-container__subtitle">Find out temperature, conditions and more...</p>
    ]);
});


