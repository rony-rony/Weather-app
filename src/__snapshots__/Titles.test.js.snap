// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Titles wrapper without crashing 1`] = `ShallowWrapper {}`;
